share/dictionary/radius/dictionary.centec

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2019 The FreeRADIUS Server project and contributors
+# This work is licensed under CC-BY version 4.0 https://creativecommons.org/licenses/by/4.0
+#
+## Centec Packet RADIUS Dictionary
+#
+# http://www.centecnetworks.com
+#
+#
+
+VENDOR		Centec				27975
+BEGIN-VENDOR	Centec
+
+#
+# Centec Privilege Attributes
+#
+ATTRIBUTE	Centec-Exec-Privilege				1	integer
```

Comment:
In the dictionaries for `v4` (_master_ branch), we don't use the prefix. therefore, please remove the `Centec-` leaving only `Exec-Privilege`